<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeepLearning_Machine</class>
 <widget class="QTabWidget" name="DeepLearning_Machine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1135</width>
    <height>883</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DeepLearning Machine</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tab_1">
   <attribute name="title">
    <string>영상분류</string>
   </attribute>
   <widget class="QLabel" name="file_dirShow">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>721</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
border:2px dotted black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QToolButton" name="data_directory">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="1.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="159">
            <red>96</red>
            <green>161</green>
            <blue>255</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="1.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="159">
            <red>96</red>
            <green>161</green>
            <blue>255</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="1.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="159">
            <red>96</red>
            <green>161</green>
            <blue>255</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="1.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="159">
            <red>96</red>
            <green>161</green>
            <blue>255</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="1.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="159">
            <red>96</red>
            <green>161</green>
            <blue>255</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="1.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="159">
            <red>96</red>
            <green>161</green>
            <blue>255</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="1.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="159">
            <red>96</red>
            <green>161</green>
            <blue>255</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="1.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="159">
            <red>96</red>
            <green>161</green>
            <blue>255</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="1.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="159">
            <red>96</red>
            <green>161</green>
            <blue>255</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(96, 161, 255, 159), stop:1 rgba(255, 255, 255, 255));
border-radius:7px;</string>
    </property>
    <property name="text">
     <string>Dataset Directory</string>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonIconOnly</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="model_teach">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Model Training</string>
    </property>
   </widget>
   <widget class="QComboBox" name="weight_comboBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>ResNet50_Weights.IMAGENET1K_V1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GoogLeNet_Weights.IMAGENET1K_V1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DenseNet169_Weights.IMAGENET1K_V1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ViT_H_14_Weights.IMAGENET1K_SWAG_E2E_V1</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>model</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="worker_spinBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>6</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="epochs_spinBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY궁서B</family>
     </font>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="lr_spinBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.001000000000000</double>
    </property>
    <property name="value">
     <double>0.001000000000000</double>
    </property>
   </widget>
   <widget class="QComboBox" name="model_comboBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>resnet50</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>googlenet</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>densenet</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>vision_transformer</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>epochs</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>learning rate</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>weight</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>worker</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="accuarcy" native="true">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>721</width>
      <height>741</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="model_teach_exit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Model Training Exit</string>
    </property>
   </widget>
   <widget class="QComboBox" name="device_comboBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>cpu</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cuda</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>device</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="shell">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>341</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="out_directory">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="1.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="159">
            <red>96</red>
            <green>161</green>
            <blue>255</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="1.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="159">
            <red>96</red>
            <green>161</green>
            <blue>255</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="1.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="159">
            <red>96</red>
            <green>161</green>
            <blue>255</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="1.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="159">
            <red>96</red>
            <green>161</green>
            <blue>255</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="1.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="159">
            <red>96</red>
            <green>161</green>
            <blue>255</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="1.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="159">
            <red>96</red>
            <green>161</green>
            <blue>255</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="1.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="159">
            <red>96</red>
            <green>161</green>
            <blue>255</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="1.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="159">
            <red>96</red>
            <green>161</green>
            <blue>255</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="1.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="159">
            <red>96</red>
            <green>161</green>
            <blue>255</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(96, 161, 255, 159), stop:1 rgba(255, 255, 255, 255));
border-radius:7px;</string>
    </property>
    <property name="text">
     <string>Output Directory</string>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonIconOnly</enum>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_2">
   <attribute name="title">
    <string>객체검출</string>
   </attribute>
  </widget>
  <widget class="QWidget" name="tab_3">
   <attribute name="title">
    <string>영상</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
